controllers/storagecluster/storageclient.go

```diff
@@ -21,6 +25,10 @@ const (
 	ocsClientConfigMapName             = "ocs-client-operator-config"
 	manageNoobaaSubKey                 = "manageNoobaaSubscription"
 	useHostNetworkForCsiControllersKey = "useHostNetworkForCsiControllers"
+	// cephNetworkAnnotationKey is the annotation key used to store network details used by ceph
+	cephNetworkAnnotationKey = "k8s.v1.cni.cncf.io/networks"
```

Comment:
I don't think this has anything to do w/ ceph, this is a cniNetworksAnnotationKey

Reply:
updated